lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Library.java

```diff
@@ -140,11 +150,11 @@ public boolean hasPatron(Patron patron) {
   }
 
   /**
-   * Return a book to the library.
+   * Check in a book from a patron.
    *
-   * @param book The book to return.
-   * @param patron The patron returning the book.
-   * @return True if the book was returned, false otherwise.
+   * @param book The book to check in.
+   * @param patron The patron to check in the book from.
+   * @return True if the book was checked in, false otherwise.
    */
   public boolean checkInBook(Book book, Patron patron) {
```

Comment:
Why keep these methods when the check in/check out functionality can be handled by `checkOutMedia` and `checkInMedia`?